scripts/badge/index.mjs

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env node
+"use strict";
```

Comment:
I think the `use strict` is not needed, specially in a `.mjs` file.

Reply:
Oh? I wasn't aware of that?